src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -81,4 +108,24 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  // public List pMessage(List messages, String user, PreparedQuery results) {
+  //   for (Entity entity : results.asIterable()) {
+  //     try {
+  //       String idString = entity.getKey().getName();
+  //       UUID id = UUID.fromString(idString);
+  //       String text = (String) entity.getProperty("text");
+  //       long timestamp = (long) entity.getProperty("timestamp");
+
+  //       Message message = new Message(id, user, text, timestamp);
+  //       messages.add(message);
+  //     } catch (Exception e) {
+  //       System.err.println("Error reading message.");
+  //       System.err.println(entity.toString());
+  //       e.printStackTrace();
+  //     }
+  //   }
+  //   return messages;
+  // }
+
```

Comment:
Seems like there is a lot of commented out code here. Please remove it because when others work on this file, they will be confused why there is commented out code here.